base/database/utils.go

```diff
@@ -70,20 +71,36 @@ func systemAdvisoriesQuery(tx *gorm.DB, accountID int) *gorm.DB {
 	return query
 }
 
+func Timestamp2Str(ts *types.Rfc3339TimestampWithZ) (*string, error) {
+	if ts == nil {
+		return nil, nil
+	}
+	ret := ts.Time().Format(time.RFC3339)
+	return &ret, nil
+}
```

Comment:
`error` is likely unnecessary as it's always `nil`

Reply:
right, I guess I had a different logic in that function and forget about it